store/sqlstore/channel_store.go

```diff
@@ -3247,14 +3302,46 @@ func (s SqlChannelStore) buildLIKEClause(term string, searchColumns string) (lik
 	return
 }
 
+func (s SqlChannelStore) buildLIKEClauseX(term string, searchColumns ...string) sq.Sqlizer {
+	// escape the special characters with *
+	likeTerm := sanitizeSearchTerm(term, "*")
+
+	if likeTerm == "" {
+		return nil
+	}
+
+	// add a placeholder at the beginning and end
+	likeTerm = "%" + likeTerm + "%"
```

Comment:
This seems like the functionality of `wildcardSearchTerm`. Any reason we are not calling that function?